flask_resources/views/base.py

```diff
@@ -30,6 +30,7 @@ class BaseView(MethodView):
         request_loader,
         url_args_parser,
         with_route,
+        error_mapper,
         error_handler,
```

Comment:
What about fusing these 2 together? Don't know that there is a big difference between the 2...

Reply:
This might work :) I will give it a try and add some tests too :)